lighthouse-treemap/app/src/main.js

```diff
@@ -77,6 +77,58 @@ class TreemapViewer {
 
     const bytes = this.wrapNodesInNewRootNode(this.depthOneNodesByGroup.scripts).resourceBytes;
     TreemapUtil.find('.lh-header--size').textContent = TreemapUtil.formatBytes(bytes);
+
+    this.createBundleSelector();
+  }
+
+  createBundleSelector() {
```

Comment:
there's a TODO on L58 about this.

Currently it does kinda bootstrap the idea of groups but then really structurally build them out in these new methods.  Feels a little odd how it does both.  I suppose if it didn't, it if wouldn't call `render()` the end of the constructor, but instead trigger the onchange for one of the two controls.